packages/sycamore-router-macro/src/route.rs

```diff
@@ -82,20 +83,26 @@ pub fn route_impl(input: DeriveInput) -> syn::Result<TokenStream> {
                 }
             }
 
-            if !has_error_handler {
+            if error_handler_name.is_none() {
                 return Err(syn::Error::new(
                     input.span(),
                     "not found route not specified",
                 ));
             }
 
+            // We implement `Default` as well here for the `Router`/`RouterBase` distinction (`Router` needs to pass a default `impl Route` to `RouterBase`)
```

Comment:
Can you move this comment next to where the impl block is? It's fine if it's in the `quote!` because it is ignored by the macro.

Reply:
Done.